### DISCUSSION TITLE
to-json Not Working

### DISCUSSION THREAD

**Discussion by dex2dot0** [postmanlabs/postman-flows#385] [Defect]
I've constructed data using a Create Data block as follows:

![image](https://user-images.githubusercontent.com/72662496/186714757-f51dcf7c-e265-42a2-83d1-303759522cbd.png)

I am then looping through each Guid and trying to send the "fields" list as a variable to my Send Request:

![image](https://user-images.githubusercontent.com/72662496/186715216-eb8b697e-ff36-494b-9f4c-8b34e01937a5.png)

Looking at the Console though, the body of my request does not convert the list to JSON:

![image](https://user-images.githubusercontent.com/72662496/186715326-241d97a6-1e41-42c6-9703-467378e7650f.png)

I also tried setting the variable outside the brackets but get the same thing:

![image](https://user-images.githubusercontent.com/72662496/186715498-a1671720-fd27-483c-bfff-869531dca6c2.png)

And finally if I try and just send the list without invoking to-json the UI gives me an error:

![image](https://user-images.githubusercontent.com/72662496/186715682-0ce490fb-e469-4495-8f84-3753d83800e2.png)

What would be the right way to pass the list as a JSON body?

**Comment by shrey-bhagat** [comment]
hey @dex2dot0 ! `to-json` won't convert the list into JSON. You can use pass the data to the send request block where you can parse the body to JSON.  : 
<img width="673" alt="Screenshot 2022-08-29 at 4 59 12 PM" src="https://user-images.githubusercontent.com/98767117/187191498-a9f50b5e-f44e-4d75-abb6-a29477666bd8.png">

**Reply by dex2dot0** [reply]
Thanks for the reply @shrey-bhagat . Gave that a try and it didn't change anything unfortunately.
